### DISCUSSION TITLE
Is the official example for to_pandas_edgelist not working?

### DISCUSSION THREAD

**Discussion by demogorgi** [networkx/networkx#5120] [Q&A] [answered]
The official example for `to_pandas_edgelist ` from https://networkx.org/documentation/stable/reference/generated/networkx.convert_matrix.to_pandas_edgelist.html
```
G = nx.MultiGraph([('A', 'B', {'cost': 1}), ('A', 'B', {'cost': 9})])
df = nx.to_pandas_edgelist(G, nodelist=['A', 'C'], edge_key='ekey')
```
throws an exception:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: to_pandas_edgelist() got an unexpected keyword argument 'edge_key'
```
I am using Python 3.6.5, Networkx 2.6.2 and Pandas 1.3.3. Is it just me, or is this a bug?

**Comment by rossbar** [comment, marked as answer]
I can't reproduce this w/ networkx 2.6.2. Are you sure that's the version you're using?

**Reply by demogorgi** [reply]
Thanks for checking. `pip list` shows me 2.6.2, but `print(nx.__version__)` returns 2.4. So actually I am using version 2.4. I use pyenv and apparently I do not know how to use it properly.

**Reply by demogorgi** [reply]
With 2.6.2 it works as expected
